components/webrtc/presentationBroadcast.js

```diff
@@ -609,7 +615,7 @@ class presentationBroadcast extends React.Component {
             switch (data.cmd) {
                 case 'gotoslide':
                     if (!that.isInitiator)
-                        changeSlide(data.data);
+                        this.changeSlide(data.data);
```

Comment:
If _this_ is available, then use it instead of _that_.

Reply:
I'm not sure why I'm using this and that over the switch case statement. There might be a reason I can't remember right now. I wouldn't want to change it in this PR. I've added a TODO instead

Reply:
Changing it to this seems to work fine, thus I'm using now this in the function instead of that.